src/plugins/vis_type_timelion/server/routes/validate_es.ts

```diff
@@ -18,43 +18,49 @@
  */
 
 import _ from 'lodash';
-import { IRouter } from 'kibana/server';
+import { IRouter, CoreSetup } from 'kibana/server';
+import { ES_SEARCH_STRATEGY } from '../../../data/server';
+import { TimelionPluginStartDeps } from '../plugin';
 
-export function validateEsRoute(router: IRouter) {
+export function validateEsRoute(router: IRouter, core: CoreSetup) {
   router.get(
     {
       path: '/api/timelion/validate/es',
       validate: false,
     },
     async function (context, request, response) {
       const uiSettings = await context.core.uiSettings.client.getAll();
-
-      const { callAsCurrentUser } = context.core.elasticsearch.legacy.client;
+      const deps = (await core.getStartServices())[1] as TimelionPluginStartDeps;
 
       const timefield = uiSettings['timelion:es.timefield'];
 
       const body = {
-        index: uiSettings['es.default_index'],
-        body: {
-          aggs: {
-            maxAgg: {
-              max: {
-                field: timefield,
+        params: {
+          index: uiSettings['es.default_index'],
+          body: {
+            aggs: {
+              maxAgg: {
+                max: {
+                  ield: timefield,
+                },
               },
-            },
-            minAgg: {
-              min: {
-                field: timefield,
+              minAgg: {
```

Comment:
Is it possible to use `AggConfigs` here? 
See `examples/search_examples/public/components/app.tsx` for an example.

@lukeelmers

Reply:
I think it's not needed here. This route used only for timelion app that will delete on 8.0.0 version. Also this agg doesn't have a type. From my point of view we can leave at it is